js/core/utils/position.js

```diff
@@ -1,9 +1,36 @@
+import $ from '../renderer';
 import config from '../config';
+import typeUtils from '../utils/type';
 
 const getDefaultAlignment = function(isRtlEnabled) {
     const rtlEnabled = isRtlEnabled ?? config().rtlEnabled;
 
     return rtlEnabled ? 'right' : 'left';
 };
 
+const getBoundingRect = (instance) => {
```

Comment:
- Do we need yet another [size utils](https://github.com/DevExpress/DevExtreme/blob/4a46c3a58e8275d4c3933506efb07c5fb4795334/js/core/utils/size.js#L37)? 
- If yes, do we need to use it [everywhere](https://github.com/DevExpress/DevExtreme/search?q=getBoundingClientRect&unscoped_q=getBoundingClientRect)?
- Can we use our [`outerWidth`](https://github.com/DevExpress/DevExtreme/blob/4a46c3a58e8275d4c3933506efb07c5fb4795334/js/core/renderer_base.js#L167) implementation?

Reply:
1) No, this size util is too specific
2) Yeah, I gonna do it
3) No, it's not fully clear how to do it from a technical point of view